docs/specs/assets.md

```diff
@@ -30,3 +30,25 @@ Even if an image asset does not have any intrinsic size, its contents MUST
 still stay within the `w`-`h` bounds when rendered.
 
 Authoring tools SHOULD export files where `w` and `h` match the physical size of the assets.
+
+<h2 id="video">Video</h2>
+
+{schema_string:assets/video/description}
+
+Video formats supported vary depending on the player. Some commonly supported formats are MP4, WebM, and MOV.
+
+{schema_object:assets/video}
+
+Video assets define the source video content that can be referenced by video frame assets.
+This enables the use of video compression techniques to reduce the overall animation size, when working with raster-based animations.
+
+Players MUST support extracting individual frames from video assets at specified timestamps when initializing animations.
+
+<h2 id="video-frame">Video Frame</h2>
+
+{schema_string:assets/video-frame/description}
+
+{schema_object:assets/video-frame}
+
+Video frame assets reference a video asset by ID and specify a timestamp to extract a specific frame. Players MUST extract the frame at the exact timestamp specified, or the closest available frame if the exact timestamp is not available. The extracted frame should be treated as a static image for rendering purposes.
```

Comment:
Can the same functionality be achieved using time remapping to lock a video layer's time?  IOW do we actually need video *frame* assets, or would video assets be sufficient?

I can think of some advantages to having a dedicated assets type, mostly on the implementation side (caching) - but there are also some downsides (more verbose than video asset + time-locked video layer).